samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -739,7 +739,7 @@
       "source": [
         "#### Set up credentials (Optional)\n",
         "\n",
-        "Setting up credentials is only required to run the custom serving container locally.\n",
+        "Setting up credentials is only required to run the custom serving container locally. To run the custom serving container, we use the credentials to download model artifacts from Google Cloud Storage buckets as how the `load` function in the custom predictor is implemented.\n",
```

Comment:
Credentials set up is required to execute the `Predictor`'s `load` function, which downloads the model artifacts from Google Cloud Storage.